README.md

```diff
@@ -1 +1,29 @@
-# New-Portfolio
\ No newline at end of file
+# New-Portfolio
+
+This is Uchechi Nwaka's Portfolio Website to showcase my projects and competencies and to improve my visibility.
+
+This project is built using
+Html
+Css and
+Figma Framework.
+
```

Comment:
- [ ]  Well done adding a README to make a professional readme file; kindly customize the sections similar to [this template](https://github.com/microverseinc/readme-template#readme), and please fill many sections as you can. To be familiar with markdown, [refer to this site](https://www.markdownguide.org/basic-syntax/)

Reply:
On it.